src/components/views/auth/Welcome.tsx

```diff
@@ -73,4 +74,8 @@ export default class Welcome extends React.PureComponent<IProps> {
             </AuthPage>
         );
     }
+
+    componentDidMount() {
+        getAnalytics().trackAnonymousEvent<IWelcomeScreenLoad>("welcome_screen_load", { foo: "bar" });
```

Comment:
why is the foo bar here? if intended then please leave a comment on it

Reply:
arg thanks for spotting this, made me realise my approach to types here is not working!